app/src/main/java/org/akvo/flow/presentation/settings/publish/PublishFilesPreferencePresenter.java

```diff
@@ -100,7 +100,7 @@ public void onNext(Long publishTime) {
                         .calculateTimeSincePublished(publishTime);
                 if (timeSincePublished < PublishedTimeHelper.MAX_PUBLISH_TIME_IN_MS) {
                     int remainingTime = publishedTimeHelper.getRemainingPublishedTime(
-                            timeSincePublished);
+                            timeSincePublished) + 1;
```

Comment:
Since we have the `PublishedTimeHelper` shouldn't the +1 be in that class instead?  Otherwise if it happens that you want remaining published time in a different class you would also need to add the +1 in that other class.  And maybe add the +1 either with a comment or as a constant with a descriptive name. 👍

Reply:
OK, you are right, I fixed it.